backend/schemas/output/excel/json/federal-awards-expended-template.json

```diff
@@ -342,36 +323,41 @@
                },
                "width": 12
             }
-         ],
+         ]
+      },
+      {
+         "header_height": 100,
+         "name": "UEI",
          "single_cells": [
             {
                "help": {
                   "link": "https://fac.gov/documentation/validation/#uei",
                   "text": "UEIs are 12 characters long and match rules as given by SAM.gov"
                },
-               "range_cell": "B2",
+               "range_cell": "A2",
                "range_name": "auditee_uei",
                "title": "Auditee UEI",
-               "title_cell": "A2",
+               "title_cell": "A1",
                "type": "single_cell",
                "validation": {
                   "custom_error": "Expecting something with only twelve characters",
                   "custom_title": "Must be length of 12",
                   "formula1": 12,
                   "operator": "equal",
                   "type": "textLength"
-               }
+               },
+               "width": 36
             },
             {
-               "formula": "=SUM(FIRSTCELLREF:LASTCELLREF)",
+               "formula": "=SUM('Form'!FIRSTCELLREF:LASTCELLREF)",
```

Comment:
Also a ticket for a future refactoring:

Is there a way we can make the name of the sheet (e.g. `Form`) a variable in the Jsonnet? (Perhaps it is.) If it isn't, we should pull those out, so it is easy to change the names of the sheets in the forms, and not break the whole doc. (But, it will break the Python... so, this is a fragile thing. Something to think about. Perhaps we can ticket that we want to think about it... the specs and schemas are becoming strong "source of truth" docs for the full stack, so thinking about how we leverage the form names might be part of that...)

Reply:
Hmm, I meant to change this but forgot. Nevertheless, I view this as essential for code maintainability, so I've now gone ahead and updated the code.